test/createNewProject.test.ts

```diff
@@ -116,6 +117,7 @@ suite('Create New Project Tests', async function (this: ISuiteCallbackContext):
     const pythonProject: string = 'PythonProject';
     test(pythonProject, async () => {
         const projectPath: string = path.join(testFolderPath, pythonProject);
+        await installFuncCoreTools(PackageManager.npm);
```

Comment:
I would actually suggest you comment this test out (and undo your changes to travis.yml) and file an issue to fix separately

The python experience is new and might be changing in the next week or two anyways. Sometimes it's easier to let it 'bake' before making tests work